app/components/header.tsx

```diff
@@ -96,24 +96,25 @@ export default function Header() {
         </Heading>
       </Link>
 
-      <Flex justifyContent="flex-end" alignItems="center" color="white" gap="5" width="100%">
```

Comment:
This change belongs to another PR and shouldn't be in this one.

Reply:
oh that's odd, why is it here...